kyuubi-main/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -101,6 +102,14 @@ trait ProcBuilder {
     file
   }
 
+  private def checkAcceptedStatus: Option[Throwable] = {
+    if (lastRowOfLog != null && lastRowOfLog.contains("state: ACCEPTED")) {
```

Comment:
This  log pattern seems bound to yarn + spark?

Shall we move it into SparkProcessBulder?

Reply:
yea, I created a issue [#668](https://github.com/NetEase/kyuubi/issues/668) to track this. This PR has not finished yet.

Reply:
sounds good